modin/experimental/core/execution/native/implementations/hdk_on_native/test/test_dataframe.py

```diff
@@ -2145,6 +2145,16 @@ def drop_rename_concat(df1, df2, lib, **kwargs):
             force_arrow_execute=True,
         )
 
+    def test_drop_row(self):
+        def drop_row(df, **kwargs):
+            return df.drop(1)
```

Comment:
We should probably have a test that goes through the top level to the lower, i.e. I think we should have a test for `.pop` method, which goes through `.drop`? Also, `.pop` is used in the issue.

Reply:
ping

Reply:
Oh, I see that you added a test for `.pop`. Thanks